openapi.yaml

```
@@ -5484,6 +5484,64 @@ paths:
+    get:
+      summary: 'Enable MCP Server'
+      description: 'Enable the MCP server endpoint at /mcp (only with root permissions).'
+      operationId: enable-mcp
+      responses:
+        '200':
+          description: 'MCP server enabled.'
+          content:
+            application/json:
+              schema:
+                properties:
+                  message: { type: string, example: 'MCP server enabled.' }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)